### ISSUE TITLE
Unable to save poses in Maya 2018.5

### ISSUE THREAD

**Issue by vigialanti** [markj3d/Red9_StudioPack#22]
Clicking the save pose button in Red8 Animation toolkit will cause maya2018.5 to freeze.

**Comment by markj3d** [comment]
This is a strange as it only ever crops up only on a few machines and I
can't re-pro it. It's all tied into a change that happened a year or so
back where we're starting to store the world matrix data in the pose for
objects and there's a call to maths that for some reason hangs. There's
actually a comment in our trunk code, around line 295 in the Red9_PoseSaver
you'll see a function called _getTransforms and a comment here:

        # for some reason this can cause a hard Maya hang!!!!
        # trying to track it down but if you experience it comment this one
line out!
        euler = map(math.degrees, rots.asEulerRotation())

just comment that out and let me know if that's still the issue.

**Comment by vigialanti** [comment]
I had to comment out the return at the end of the function also, because euler was undefined and threw up an error.

        #euler = map(math.degrees, rots.asEulerRotation())

        #return {'translation': [trans.x, trans.y, trans.z],
        #       'quaternion': [rots.x, rots.y, rots.z, rots.w],
        #        'euler': euler}

I'm not sure if anything else is dependent on the return but poses will save...

**Comment by markj3d** [comment]
yeah I've seen this on a few machines and I've no idea why that standard Python call would crash maya so badly.....
